tsdb/executor_test.go

```diff
@@ -288,7 +288,7 @@ func TestWritePointsAndExecuteTwoShardsQueryRewrite(t *testing.T) {
 		expected  string // Expected results, rendered as a string
 	}{
 		{
-			stmt:     `SELECT * FROM cpu`,
+			stmt:     `SELECT * FROM cpu GROUP BY *`,
```

Comment:
Yes, per the `todo` checkbox in the PR :-).

I probably have a day or so of adding the tests that don't have `GROUP BY`.  If we don't do that, there is a chance we can easily break in the future due to how the code branches.